tools/msh2vtu.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="msh2vtu" name="GMSH_To_VTU_Converter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
```

Comment:
name should not have undescores - just use spaces. But, my suggestion would be `ogstools msh2vtu`.

`python_template_version` is not needed, please remove.